test/over_react_redux/connect_test.dart

```diff
@@ -417,6 +418,42 @@ main() {
           expect(methodsCalled, contains('areStatesEqual'));
           expect(jacket.mountNode.innerHtml, contains('Count: 0'));
         });
+
+        group('when not specified,', () {
+          // This indirectly tests the memoization of wrapInteropValue
+          test('does not rerender when the state object is identical', () async {
```

Comment:
I  made this separate since I could see additional tests going in this block, and I wanted this one test to be split out and have that comment above it.